aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -425,17 +415,12 @@ impl AptosVM {
             WriteSetPayload::Direct(change_set) => change_set.clone(),
             WriteSetPayload::Script { script, execute_as } => {
                 let mut tmp_session = self.0.new_session(storage, session_id);
-                let aptos_version = self.0.get_version().map_err(Err)?;
                 let senders = match txn_sender {
                     None => vec![*execute_as],
                     Some(sender) => vec![sender, *execute_as],
                 };
-                let remapped_script =
-                    if aptos_version < aptos_types::on_chain_config::APTOS_VERSION_2 {
-                        None
-                    } else {
-                        script_to_script_function::remapping(script.code())
-                    };
+                let remapped_script = script_to_script_function::remapping(script.code());
```

Comment:
fun thing is, we don't have remapped scripts any more :D, but no need for this pr to fix that